### PR TITLE
Update doc, include testing instruction

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +69,22 @@ USER=myuser make image push
...
+E.g:
+
+```bash
+IMAGE=quay.io/${USER}/plugin:pr-xx make image push
+
+oc edit FlowCollector cluster
+# Here replace image with the newly created one under .spec.consolePlugin.image
+```
+
+If you had previously used the console with the plugin installed, you may need to restart console pods to clear cache:
```
Comment (jotak) [inline README.md:86]: not sure if this is 100% accurate (I don't know exactly what is cached and what isn't), but ... it worked for me :)

**Review by mariomac — APPROVED** [review]
/lgtm

**Review by jpinsonneau — APPROVED** [review]
:+1:

**Pull request merged**